cirq-superstaq/cirq_superstaq/service.py

```diff
@@ -792,6 +792,74 @@ def process_dfe(self, ids: List[str]) -> float:
         """
         return self._client.process_dfe(ids)
 
+    def submit_aces(
+        self,
+        target: str,
+        qubits: List[int],
+        shots: int,
+        num_circuits: int,
+        mirror_depth: int,
+        extra_depth: int,
+        method: Optional[str] = None,
+        noise: Optional[Union[str, cirq.NoiseModel]] = None,
+        error_prob: Optional[Union[float, Tuple[float, float, float]]] = None,
+        tag: Optional[str] = None,
+        lifespan: Optional[int] = None,
+    ) -> str:
+        """Performs a POST request on the `/aces` endpoint.
```

Comment:
imo this docstring is too low-level for a user-facing function - it should be more descriptive (and probably include a link to the ACES paper)

(ditto for functions in `gss.Service` and `qss.SuperstaqBackend`)

Reply:
Updated the docstring. Let me know if the new level of detail looks good.